guides/common/modules/proc_adding-custom-deb-repositories.adoc

```diff
@@ -0,0 +1,72 @@
+[id="Adding_Custom_DEB_Repositories_{context}"]
+= Adding {customdebtitle} Repositories
+
+Use this procedure to add {customdeb} repositories in {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-custom-deb-repositories[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
+. In the *Name* field, enter a name for the repository.
+{Project} automatically completes the *Label* field based on what you have entered for *Name*.
+. Optional: In the *Description* field, enter a description for the repository.
+. From the *Type* list, select `deb` as type of repository.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+You can find the upstream URLs on Debian-based systems in `/etc/apt/sources.list`.
+. Optional: In the *Releases* field, set a or multiple releases.
++
+For official Debian repositories, {Team} recommends setting a _codename_ in the *Releases* field, for example `buster` for Debian 10 or `bullseye` for Debian 11.
+Do not use `stable` or `testing` as they merely reference a currently corresponding _codename_.
```

Comment:
I don't understand what you're trying to say here if I am honest.

Reply:
So Debian Bullseye is always Debian 11;  but Debian stable is currrently Debian 11, but will be Debian 12 with the next release.

Reworded.